test/e2e/pod_top_test.go

```diff
@@ -119,13 +120,15 @@ var _ = Describe("Podman top", func() {
 		_, ec, podid := podmanTest.CreatePod("")
 		Expect(ec).To(Equal(0))
 
-		session := podmanTest.Podman([]string{"run", "-d", "--pod", podid, ALPINE, "top", "-d", "2"})
+		session := podmanTest.Podman([]string{"run", "-d", "--pod", podid, ALPINE, "sleep", "2000"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 
-		session = podmanTest.Podman([]string{"run", "-d", "--pod", podid, ALPINE, "top", "-d", "2"})
+		session = podmanTest.Podman([]string{"run", "-d", "--pod", podid, ALPINE, "sleep", "2000"})
```

Comment:
It seems there was a race condition between the two seconds sleep of `top` and `podman pod top`. If top exited earlier, the tests would fail. Are the pods stopped/killed after the test?